src/PySXO/core/request_handler.py

```diff
@@ -28,69 +33,54 @@ def __init__(self, client_id, client_password, cache, dry_run):
             'limit': 100
         }
     
-    def _get(self, **kwargs):
+    def _get(self, **kwargs) -> Union[List, Dict]:
         LOGGER.info('Invoking _get function')
         return self._request(method='get', **kwargs)
     
-    def _post(self, **kwargs):
+    def _post(self, **kwargs) -> Union[List, Dict]:
         LOGGER.info('Invoking _post function')
         return self._request(method='post', **kwargs)
-    
-    def _request(self, method='get', paginated=False, uri=URI, **kwargs):
-        LOGGER.info('Invoking _request function:')
-        LOGGER.debug(f'\tMethod: {method}\n\tPaginated: {paginated}\n\tURI: {uri}\n\tKwargs:{kwargs}')
+
+    def _request(self, method: str = 'get', uri: str = URI, **kwargs) -> Union[List, Dict]:
+        
         if method != 'get' and self.dry_run:
             return {}
-        # refresh jwt
+
         kwargs['headers'] = {**self.headers, **kwargs.get('headers', {})}
-        kwargs['params'] = {**self.params, **kwargs.get('params', {})}
         kwargs['url'] = f'{RequestHandler.BASE_URL}{uri}{kwargs["url"]}'
-        LOGGER.info('Sending request')
-        result = requests.request(method=method, **kwargs)
-        LOGGER.info('Got response')
+        
+        LOGGER.debug(json.dumps(dict(kwargs)))
```

Comment:
Thats nice but I want to know what is it when I see it in logs, add some context

Reply:
agree with @Drrechu here. Just a couple words that let me know these are the kwargs would be helpful

Reply:
Dropping the log totally